### DISCUSSION TITLE
Problem in building provider and consumer connectors

### DISCUSSION THREAD

**Discussion by Akshay-a-j** [eclipse-edc/Samples#72] [General]
I’m having trouble building provider and consumer connectors from Samples/transfer/transfer-06-consumer-pull-http. I get these errors when I run the command:
 java -Dedc.keystore=transfer/transfer-06-consumer-pull-http/certs/cert.pfx -Dedc.keystore.password=123456 -Dedc.vault=transfer/transfer-06-consumer-pull-http/http-pull-provider/provider-vault.properties -Dedc.fs.config=transfer/transfer-06-consumer-pull-http/http-pull-provider/provider-configuration.properties -jar transfer/transfer-06-consumer-pull-http/http-pull-connector/build/libs/pull-connector.jar

 Error: Could not find or load main class .keystore=transfer.transfer-06-consumer-pull-http.certs.cert.pfx 
 Caused by: java.lang.ClassNotFoundException: /keystore=transfer/transfer-06-consumer-pull-http/certs/cert/pfx

Does anyone know how to fix this?

**Comment by carlos-schmidt** [comment]
If you are using PowerShell, you can try this:
`java "-Dedc.keystore=transfer/transfer-06-consumer-pull-http/certs/cert.pfx" "-Dedc.keystore.password=123456" "-Dedc.vault=transfer/transfer-06-consumer-pull-http/http-pull-provider/provider-vault.properties" "-Dedc.fs.config=transfer/transfer-06-consumer-pull-http/http-pull-provider/provider-configuration.properties" -jar transfer/transfer-06-consumer-pull-http/http-pull-connector/build/libs/pull-connector.jar`

So just putting all the -D options in quotes

**Reply by Akshay-a-j** [reply]
Thank you @carlos-schmidt. Its working now.

**Comment by Akshay-a-j** [comment]
It runs fine if I remove arguments and execute just jar file. But, to run two connectors on the same system I will have to pass config.properties to each connector. That is when the above errors are being thrown.
